atom/browser/browser.h

```diff
@@ -29,6 +29,22 @@ namespace gfx {
 class Image;
 }
 
+#if defined(OS_WIN)
+namespace atom {
+enum class JumpListResult : int;
```

Comment:
Since the `SetJumpList` is a public API of `Browser`, the `JumpListResult` should also be public too. So the complete definition of `JumpListResult` should be placed in `browser.h`.